KosherCocoa/Library/Core/Calendar/Chagim/KCJewishCalendar.m

```diff
@@ -193,41 +199,45 @@ - (NSInteger)yomTovIndex
             if (![self isCurrentlyHebrewLeapYear])
             {
                 // if 13th Adar falls on Friday or Shabbos, push back to Thursday
-                if ((([self currentHebrewDayOfMonth] == 11 || [self currentHebrewDayOfMonth] == 12)&& [self currentDayOfTheWeek] == 5)
-                    || ([self currentHebrewDayOfMonth] == 13 && !([self currentDayOfTheWeek] == 6 || [self currentDayOfTheWeek] == 7)))
+                if (((day == 11 || day == 12) && dayOfWeek == 5)
+                    || (day == 13 && !(dayOfWeek == 6 || dayOfWeek == 7)))
                 {
                     return kFastOfEsther;
                 }
-                if ([self currentHebrewDayOfMonth] == 14)
+                if (day == 14)
                 {
                     return kPurim;
                 }
-                else if ([self currentHebrewDayOfMonth] == 15)
+                else if (day == 15)
                 {
                     return kShushanPurim;
                 }
             }
             // else if a leap year
             else
             {
-                if ([self currentHebrewDayOfMonth] == 14)
+                if (day == 14)
                 {
                     return kPurimKatan;
                 }
+                if (day == 15)
+                {
+                    return kShushanPurimKatan;
```

Comment:
added shushan purim katan

Reply:
Is this a thing? Today I learned!

Reply:
It was recently added to KosherJava, I just followed